truffle/migrations/1_initial_migration.js

```diff
@@ -3,23 +3,18 @@ require('dotenv').config({ path: `${process.env.PWD}/.env` });
 const { waitForAccountToUnlock, extractEnvAccountAndPwd } = require('../test/utils');
 const Migrations = artifacts.require("./Migrations.sol");
 
-module.exports = (deployer) => {
+module.exports = async (deployer) => {
   process.env.NETWORK = deployer.network;
   const { from, pwd } = extractEnvAccountAndPwd(deployer.network);
 
-  let isUnlock = web3.personal.unlockAccount(from, pwd, 1000);
+  let isUnlock = await web3.eth.personal.unlockAccount(from, pwd, 1000);
   if (!isUnlock) {
     console.error(`Account ${from} could not be unlock`);
     process.exit(1);
   }
 
-  // waitForAccountToUnlock(from).then(() => {
   console.log(`Account ${from} was unlocked successfully.`);
   console.log("Deploying `Migrations.sol` ...");
-  deployer.deploy(Migrations, { overwrite: false }).then(() => {
```

Comment:
It was useful to know when a migration deployment was completed. Why did you remove this?

Reply:
hm ok, i will add it, it wasn't necessary before as there is plenty of logs outputed already but given the circumstanes how badly Truffle runs them and all the race conditions i will put it back.